analyses/summary/helpers.go

```diff
@@ -13,18 +15,19 @@ import (
 )
 
 // ProcessSummary processes summary results, return true if Vul found
-func ProcessSummary(analysedResult driver.GetResponseType, jsonOut bool, showVerboseMsg bool) bool {
+func ProcessSummary(ctx context.Context, analysedResult *driver.GetResponseType, jsonOut bool, showVerboseMsg bool) bool {
 	out := getResultSummary(analysedResult)
 	if jsonOut {
 		outputSummaryJSON(out)
 	} else {
 		outputSummaryPlain(out, showVerboseMsg)
 	}
+	telemetry.SetVulnerability(ctx, out.TotalVulnerabilities)
```

Comment:
Instead of spreading telemetry call all over the place, can this function return the `out` to the caller and add count the place where all other properties added.

Reply:
I feel that's okay to spread the telemetry package and context (ctx) across. 

You can suppose a case of adding an extra property that we calculate in `cmd-> controller->pkg->Analyse` .. are we supposed to edit each one of redirecting function to update its return value, then finally setting `ctx` in cmd package..?
I'm not sure that's a good idea.

https://github.com/code-ready/crc/search?q=telemetry.Set

Reply:
but we are not in that situation, anyways, I let the other reviewers to take the final decision.